src/api/app/views/webui/project/subprojects.html.haml

```diff
@@ -10,12 +10,12 @@
     %thead
       %tr
         %th Parent project
-        %th Description
+        %th Title
     %tbody
       - @parentprojects.each do |project|
         %tr
-          %td= link_to project.name, :action => 'show', :project => project.name
-          %td= project.title
+          %td{:width => "50%"}= link_to project.name, :action => 'show', :project => project.name
```

Comment:
ok, I read that now - but I question that 50% is a good choice between name fraction and title